semgrep-core/src/core/Semgrep_error_code.ml

```diff
@@ -99,7 +99,10 @@ let exn_to_error ?(rule_id = None) file exn =
       mk_error ~rule_id loc msg Timeout
   | Out_of_memory ->
       let loc = Parse_info.first_loc_of_file file in
-      mk_error ~rule_id loc "Out of memory" OutOfMemory
+      mk_error ~rule_id loc "Heap space exceeded" OutOfMemory
```

Comment:
Just curious about this change, isn't "out of memory" a rather standard way of reporting this?

Reply:
The error message will now say "Out of memory: heap space exceeded" (vs if you exceed your own memory limit it will say "Out of memory: memory limit reached").